conans/client/api/conan_api.py

```diff
@@ -165,6 +166,28 @@ def user_remove(self, remote_name):
     def user_update(self, user_name, user_pasword):
         return {}
 
+    @api_method
+    def get_active_remotes(self, remote_names):
+        remotes = self.app.cache.registry.load_remotes()
+        all_remotes = remotes.all_values()
+
+        if not all_remotes:
+            raise ConanException("The remotes registry is empty. "
+                                 "Please add at least one valid remote.")
+
+        # If no remote is specified, search in all of them
+        if not remote_names:
+            # Exclude disabled remotes
+            all_remotes = [remote for remote in all_remotes if not remote.disabled]
+            return all_remotes
```

Comment:
This seems to be leaking undocumented classes via the API. So far, the goal was that the API both received and returned plain/primitive python objects. Here it is returning a "Remote" object, that we should make part of the public API too, documented, and commit to stability.

Again, API design, to be discussed separately, not this PR.

Reply:
Yes, this Remote class should be documented, because it's extensively used.

Reply:
But one thing is the programming interface and a much different one is the API one.
If we propagate this object through the API, then we will be exposing all its public interface, and even if we restrict it in documentation, we could have a stability problem. It could make sense to return through the api simplified objects, proxy wrappers, or something that could protect the internal programming interface.